src/main/java/com/coherent/training/api/kapitsa/clients/Authenticator.java

```diff
@@ -5,26 +5,27 @@
 import com.coherent.training.api.kapitsa.util.plainobjects.Token;
 import lombok.SneakyThrows;
 import org.apache.http.NameValuePair;
+import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.http.message.BasicNameValuePair;
 import org.slf4j.LoggerFactory;
 
-import java.util.ArrayList;
-import java.util.Base64;
-import java.util.List;
+import java.util.*;
 
+import static com.coherent.training.api.kapitsa.base.BaseClientObject.BaseClientObjectBuilder;
 import static com.coherent.training.api.kapitsa.providers.UrlProvider.OAUTH_URL;
 import static com.coherent.training.api.kapitsa.util.DataHandler.getTokenObj;
 import static org.apache.http.HttpHeaders.AUTHORIZATION;
 import static org.apache.http.HttpHeaders.CONTENT_TYPE;
 import static org.apache.http.HttpStatus.SC_OK;
 
-public class Authenticator extends BaseClientObject {
+public class Authenticator {
     private static final String CLIENT_ID = ConfigFileReader.getInstance().getClientId();
     private static final String CLIENT_SECRET = ConfigFileReader.getInstance().getClientSecret();
     private static final String oauthUrl = OAUTH_URL.getEndpoint();
```

Comment:
do not see any config in the repo

Reply:
https://github.com/roma0593/APItraining/tree/master/configs